frodo.js

```diff
@@ -2,67 +2,70 @@
  * Extras Plugin
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!